File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/spot/SpotCancelBatchOrderReq.java
package com.bitget.openapi.dto.request.spot;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.io.Serializable;
import java.util.List;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Batch cancellation request
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SpotCancelBatchOrderReq implements Serializable {

    private static final long serialVersionUID = -1L;

    /**
     * Currency pair
     */
    private String symbol;

    /**
     *  Order ids
     */
    private List<String> orderIds;
}

File: bitget-node-sdk-api/src/lib/model/spot/order/SpotCancelBatchOrderReq.ts
export class SpotCancelBatchOrderReq {
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     *  Order ids
     */
    private _orderIds!:string[];


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderIds(): string[] {
        return this._orderIds;
    }

    set orderIds(value: string[]) {
        this._orderIds = value;
    }
}